clicache/src/geode_defs.hpp

```diff
@@ -33,6 +33,11 @@
 // Disable native code generation warning
 #pragma warning(disable: 4793)
 
+// Disable Native Conditional Unique warning
+#pragma warning(disable: 4383)
+
+// C4103alignment changed after including header, may be due to missing #pragma pack(pop)
+#pragma warning(disable: 4103)
```

Comment:
Are these changes necessary to relevant to the immediate issue being addressed?

Reply:
These two warnings were causing over 1000 warnings, so I could not see the warnings I was generating. I took them out as they are CLI related, but not real issues.